data/src/error_analysis/README.md

```diff
@@ -0,0 +1,45 @@
+# Error Analysis tool for G2P
+
+This tool could be used for performing fine-grain error analysis of a G2P model. It reports a performance matrix of a test output generated by a G2P model. 
```

Comment:
spell "G2P" out as "grapheme-to-phoneme conversion" the first time you mention it. E.g.,

```This tool performs fine-grained error analysis of a grapheme-to-phoneme conversion (G2P) model..."

I think it should be "fine-grained" and the term "performance matrix" is novel to me---is it the same or different than a confusion matrix?

I have also suggested "performs" instead of "could be used for performing" just to keep things briefer.

Reply:
Performance matrix is the same as the confusion matrix. In the paper, I used performance matrix and kept the same here. I hope this name makes sense in this context.